pkg/networkmanager/dialogs-common.jsx

```diff
@@ -198,10 +201,27 @@ export const NetworkAction = ({ buttonText, iface, connectionSettings, type }) =
         if (type == 'vlan') settings = getVlanGhostSettings();
         if (type == 'team') settings = getTeamGhostSettings({ newIfaceName });
         if (type == 'bridge') settings = getBridgeGhostSettings({ newIfaceName });
+        if (type == 'wireguard') settings = getWireGuardGhostSettings({ newIfaceName });
     }
 
     const properties = { connection: con, dev, settings };
 
+    async function resolveDepsAndShowDialog(dlg) {
+        try {
+            await cockpit.script("command -v wg");
+            Dialogs.show(dlg);
+        } catch {
+            try {
+                const config = cockpit.manifests.network.config;
+                const os_release = await read_os_release();
+                await install_dialog(config.wireguard_package[os_release.ID]);
+                Dialogs.show(dlg);
+            } catch (e) {
+                console.error(e);
```

Comment:
If this just becomes error handling for `command -v wg`, that's fine -- that really Should Not Fail™. But if it's error handling for the on-demand install or wg dialog, that needs to become proper Alerts and shown in the UI.